### ISSUE TITLE
Website updated and addon stopped working

### ISSUE THREAD

**Issue by ExperTnot** [dteviot/WebToEpub#953]
**Describe the bug**
When I try to use the addon on the site "https://lightnovelstranslations.com/novel/i-woke-up-piloting-the-strongest-starship-so-i-became-a-space-mercenary/?tab=table_contents"
 I get an error, which I didnt get before. They updated their site and I am guessing that is the reason why it stopped working but I am not sure so I submitted this report. Also since I dont know any technical stuff when i tried googling the messages i gave up. Here is the error message (also in attached screenshot): 
"TypeError: Cannot read properties of null (reading 'cloneNode')
    at LightNovelsTranslationsParser.getChapterUrls (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/parsers/WordpressBaseParser.js:37:44)
    at LightNovelsTranslationsParser.getChapterUrls (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/parsers/LightNovelsTranslationsParser.js:13:26)
    at LightNovelsTranslationsParser.onLoadFirstPage (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/Parser.js:378:14)
    at processInitialHtml (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/main.js:47:24)
    at populateControlsWithDom (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/main.js:252:9)
    at onMessageListener (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/main.js:16:13)" 
After I click ok it puts some stuff in the addon but not how it usually works(see screenshot).

**To Reproduce**
1. Go to '(https://lightnovelstranslations.com/novel/i-woke-up-piloting-the-strongest-starship-so-i-became-a-space-mercenary/?tab=table_contents)'
2. Click on 'WebToEpub addon'
3. Optional: Click OK button

**Expected behavior**
No error code and normal filling in of the epub chapters, info, etc.

**Screenshots**
After trying to use WebToEpub addon
![After trying to use WebToEpub addon](https://user-images.githubusercontent.com/130067455/230395200-7a4bdb66-8d1d-48d1-b37f-6f31211e8d55.png)
After clicking OK button
![After OK button](https://user-images.githubusercontent.com/130067455/230395337-c516302e-553d-4f11-a3a0-5e4662a0e93a.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version: 112.0.5615.50

**Smartphone (please complete the following information):**
Was on Browser.

**Additional context**
Im sorry if this is just becasue the site changed and i should have just submitted the site again but since the site was already added (before it updated) I made this report.

**Comment by dteviot** [comment]
@ExperTnot 
Currently I have RL issues, so no time.  Might be a few weeks before I can look into this.

**Comment by ExperTnot** [comment]
Ok no problem. I can read on their website for now just would like to read offline if possible. Also didn’t say this but thanks for your work so far. :) Should I comment here again in like 3 weeks to remind you?

**Comment by dteviot** [comment]
@ExperTnot 
>Should I comment here again in like 3 weeks to remind you?

No.  The whole point of this issue tracking system is so that I don't forget things.

**Comment by ExperTnot** [comment]
Ok I see. Never really used github except downloading stuff. I guess I'll get a notification if this thread updates. Thanks in advance!

**Comment by dteviot** [comment]
Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://lightnovelstranslations.com/novel/i-woke-up-piloting-the-strongest-starship-so-i-became-a-space-mercenary/?tab=table_contents, chapter 1 and 401
* https://lightnovelstranslations.com/novel/welcome-to-the-impregnable-demon-king-castle-the-black-sorcerer-is-offered-a-seat-as-a-member-of-the-demon-king-armys-top-brass-after-being-kicked-out-of-the-heros-party-because-they-didn/ chapter 1

Note, I did not attempt to collect the images.  (e.g. stuff under the "Novel Illustrations" tab.  It's all lazy loaded, and different format to rest of pages.  Why didn't they do like everyone else, and just put the images in their own chapter?)

For my notes: 43 minutes work
